managing-os/templates.md

```diff
@@ -87,6 +87,17 @@ Important Notes
 
        $ sudo yum remove qubes-template-fedora-24
 
+ * Standalone VMs using Template VMs as a basis can be created easily. These
+   VMs receive a *copy* of the operating system and do not get automatically
+   updated when Template VMs are updated--they must be updated individually.
+   
+ * Template VMs can occupy more space on the dom0 filesystem than necessary
+   because they cannot employ automatic TRIM/discard on the root fs. The
+   `qvm-trim-template` command in dom0 is used to recover this unused space.
```

Comment:
Ah, well I suppose it is true regardless because we do not (yet?) enable discard on / at all. My question was incorrectly framed with the assumption that that change had been made. So nevermind :) We'll just need to remember to revisit this if that changes.